app/test_main.py

```diff
@@ -16,6 +16,18 @@ class TestGetHumanAge:
             (28, 28, [3, 2]),
             (100, 100, [21, 17]),
         ],
+        ids=[
+            "Should be zeros if years equal to 0",
+            "Should be zeros if years less than 15",
+            "Should be ones if years equal to 15",
+            "Should be ones if years less than 24",
+            "Should be twos if years equal to 24",
+            "Should be twos if years less than 28",
+            "Should be three for cat years and two for dog years "
+            "if years equal to 28",
+            "Should be 21 for cat years and 17 dog years "
+            "if years equal to 100",
+        ]
```

Comment:
I meant not just mentioning it, but explaining why, like: "Newborn cats and dogs should be equivalent to 0 human years"
Well, still good job!